src/Tools/ExternalAccess/Xamarin/Microsoft.CodeAnalysis.ExternalAccess.Xamarin.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.Xamarin</RootNamespace>
+    <TargetFramework>net472</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.TypeScript</PackageId>
+    <PackageDescription>
+      A supporting package for Xamarin:
+      https://github.com/xamarin/CodeAnalysis
+    </PackageDescription>
+    
+    <XamarinKey Condition="'$(XamarinKey)' == ''">002400000480000094000000060200000024000052534131000400000100010079159977d2d03a8e6bea7a2e74e8d1afcc93e8851974952bb480a12c9134474d04062447c37e0e68c080536fcf3c3fbe2ff9c979ce998475e506e8ce82dd5b0f350dc10e93bf2eeecf874b24770c5081dbea7447fddafa277b22de47d6ffea449674a4f9fccf84d15069089380284dbdd35f46cdff12a1bd78e4ef0065d016df</XamarinKey>
```

Comment:
❗️ This belongs in this block:

https://github.com/dotnet/roslyn/blob/c4c3e1ab4a47d847359b2ff82e70bcb737909e0b/eng/targets/Settings.props#L87-L98